lib/src/chip/chip-row/chip-row.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+import { ChipComponent } from '../chip.component';
+import { anyToBoolean } from '../../util';
+
+@Component({
+    selector: 'hc-chip-row',
+    template: `<div [ngClass]="contentClass"><div><div class="row-buffer"><ng-content></ng-content></div></div></div>`,
```

Comment:
I think you could remove the getter/setter, and also get rid of `contentClass` and do this:
```
class="chip-row-contents" [ngClass]="{'single-row': !wrap}"
```